manager-bundle/skeleton/bin/console

```diff
@@ -19,6 +19,15 @@ set_time_limit(0);
 @ini_set('zlib.output_compression', '0');
 @ini_set('display_errors', '0');
 
+// System maintenance mode comes first as it has to work even if the vendor directory does not exist
+if (
+    'contao:cron' === ($_SERVER['argv'][1] ?? null)
+    && file_exists(__DIR__.'/../var/maintenance.html')
+) {
+    echo 'Contao maintenance mode is enabled, skipping cron jobs.';
+    exit(-1);
```

Comment:
If we go ahead with this, we should probably not echo anything and also exit with `0` - otherwise you will get minutely emails if `MAILTO` is set in your crontab.

Reply:
changed in 7c05a9dfd7cb221935656b0f3e48f56913590488